packages/components/src/components/StepIndicator/StepIndicator.tsx

```diff
@@ -27,7 +27,7 @@ const StepIndicator = React.forwardRef<HTMLOListElement, StepIndicatorProps>(
         {
           // eslint-disable-next-line lodash/prefer-lodash-method
           steps.map((step, index) => (
-            <StepIndicatorSegment isActiveStep={current === index} key={index}>
+            <StepIndicatorSegment isActiveStep={current >= index} key={index}>
```

Comment:
The `isActiveStep` prop name is misleading now since we're highlighting up to the current step.

Reply:
@RohitRajendran good catch. That also messes up the aria-current information as well. Probably should add a `completed` prop instead.